### PR TITLE
Various fixes and cleanups

### LINKED ISSUE #53: Middleware should not throw exception when origin was not allowed
### What did you do?

Made a POST request from a not allowed origin

### What did you expect to happen?

The post request to succeed and a CORS error in the browser console

### What actually happened?

Exception

### PR DESCRIPTION
**Pull request by Jalle19**
Fixes #53

Changes proposed in this pull request:

 * stop tracking some files that shouldn't be tracked
 * change how non-preflight CORS requests are handled (#53)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea
 /vendor/
+composer.lock
```
Comment (mathewrapid) [inline .gitignore:3]: Ha, good one 😄

File: src/CorsServiceProvider.php [hunk 2]
```diff
@@ -20,7 +20,6 @@ public function register()
         }
 
         $this->registerBindings();
-        $this->registerFacades();
```
Comment (mathewrapid) [inline src/CorsServiceProvider.php:23]: I have no idea how the facades worked. Is this a BC-break?

File: src/CorsService.php [hunk 3]
```diff
@@ -105,12 +105,6 @@ public function handlePreflightRequest(Request $request)
      */
     public function handleRequest(Request $request, Closure $next)
     {
-        try {
-            $this->validateRequest($request);
```
Comment (mathewrapid) [inline src/CorsService.php:109]: 1. I the `validateRequest(Request $request)` used anywhere anymore?
2. This is a BC-break

File: src/CorsServiceProvider.php (hunk 2 above)
Comment (Jalle19) [reply src/CorsServiceProvider.php:23]: Yes, bumping to 3.0.0 anyway

**Comment by Jalle19** [comment]
@mathewrapid regarding BC stuff it's fine, gonna bump the major version anyway

File: src/CorsService.php (hunk 3 above)
Comment (Jalle19) [reply src/CorsService.php:109]: Removed it

**Pull request merged**